dali/c_api/c_api_test.cc

```diff
@@ -251,6 +250,7 @@ TYPED_TEST(CApiTest, FileReaderPipe) {
   pipe_ptr->RunGPU();
 
   ComparePipelinesOutputs<TypeParam>(handle, *pipe_ptr);
+  ws.Clear();
```

Comment:
Why we need this call here?

Reply:
Like above, it originated when the streams were created and destroyed, not taken from the pool.

Reply:
So the code that uses DALI C API needs to call `Clear` on created ws explicitly?